commands/make.js

```diff
@@ -25,7 +25,13 @@
 			commands = {
 				controller: function() {
 					var name = config.args.name || '',
-						target = path.join(sourcePath, 'js/build/' + slugify(name) + '.js');
+						type = config.args.type || 'base';
+
+					if (type !== 'base') {
+						name = type;
+					}
+
+					var target = path.join(sourcePath, 'js/build/' + slugify(name) + '.js');
```

Comment:
Declare the 'target' variable at the top of this function with the rest of the variable declarations, and then just define it here to account for variable hoisting.